code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -10,6 +10,27 @@
 	//Eye colour
 	var/eyes_color = "#000000"
 
+	//plug untill baymed
+	//Hair colour and style
+	var/r_hair = 0
+	var/g_hair = 0
+	var/b_hair = 0
+	//Facial hair colour and style
+	var/r_facial = 0
+	var/g_facial = 0
+	var/b_facial = 0
+	//Eye colour
+	var/r_eyes = 0
+	var/g_eyes = 0
+	var/b_eyes = 0
+	//Skin colour
+	var/r_skin = 0
+	var/g_skin = 0
+	var/b_skin = 0
```

Comment:
All that vars was removed by us and replaced with solid one.
```
 	//Hair colour and style
 	var/hair_color = "#000000"
 	var/h_style = "Bald"
 
 	//Facial hair colour and style
 	var/facial_color = "#000000"
 	var/f_style = "Shaved"
 
	//Eye colour
	var/eyes_color = "#000000
```

Reply:
We will have to revisit this after baymed, there is no point to fix it now

Reply:
???
That's unused vars.
If them using now that will break something.
We should remove it or fix for now.

Reply:
Entire mob folder will be replaced in baymed no point to fix it now

Reply:
Why vars was add withing that PR?

Reply:
To remove need to add it later after baymed. Otherwise it can be forgotten and adding them now dont bring anyharm

Reply:
But we had removed them and replaced already. That's step back and will required rewriting of external organs code, preference handling and much more things.

Reply:
You are right, at first i thought you dont have them at all

Reply:
ill fix it tomorrow